### DISCUSSION TITLE
Webpage theme

### DISCUSSION THREAD

**Discussion by arijitdas123student** [arijitdas123student/kartben-artificial-nose#3] [Ideas]
@kartben How about a theme like this: https://html5up.net/landed ?
Lemme know if you find it to be interesting.....I will customize as per the project!

**Comment by kartben** [comment]
@arijitdas123student nice find! It is quite good indeed -- let's start with this one!

**Reply by arijitdas123student** [reply]
Awesome!

**Reply by kartben** [reply]
should we move the wiki page I started to /discussions? And do you have thoughts on what other top level "blocks" we should include? Maybe "Press" to link to published articles, videos, etc.?

**Reply by arijitdas123student** [reply]
Yeah I mean discussions are good...and I think we can even add the guide links(e.g. hackster) and more videos!

**Reply by arijitdas123student** [reply]
tweets can be added as well

**Reply by kartben** [reply]
yepppp

**Reply by arijitdas123student** [reply]
What should I add as a brief overview of about 100-150 words ?

**Reply by kartben** [reply]
Maybe have something along the lines of this quote?

> Back in May 2020, in the early days of the pandemic lockdown situation, I was spending time trying to perfect my bread recipe, including trying to automatically determine when my sourdough starter would be in the ideal condition to bake perfect baguettes. 
> This soon turned into the perfect excuse to learn about gas sensors, neural networks, and to eventually publish the full instruction set for a DIY, low-cost, artificial nose. Everyone, meet Nosetradamus!

**Reply by arijitdas123student** [reply]
Nice....but I guess we can write that in other places....like for a larger explanation....but is there anything that you can fit in a couple of lines?
Plus can I write "Nosetradamus" as the main project name?

**Reply by kartben** [reply]
> Nice....but I guess we can write that in other places....like for a larger explanation....but is there anything that you can fit in a couple of lines?

Ya, let me think about it... How about:

> Nosetradamus is an open-source and open-hardware artifical nose. Teach it the scents you care about, and add the sense of smell to your next application.

> Plus can I write "Nosetradamus" as the main project name?

Yes. Would probably be cool to design some kind of logo around that nickname, at some point :) FWIW I have http://nosetradam.us registered :)

**Reply by arijitdas123student** [reply]
Awesome....that's great....so we can just push this repo up to http://nosetradam.us using GH Pages! ;)

**Reply by arijitdas123student** [reply]
Okay so for testing the website is live here : https://arijitdas.me/kartben-artificial-nose/

**Reply by kartben** [reply]
Nice! I like the favicon 😄 👃🏼

**Reply by arijitdas123student** [reply]
Haha found nothing else than that! ;D
